### PR TITLE
Feature/mvp application

### PR DESCRIPTION
**Pull request by ahmed-ae**
MVP Application submission form

## Description
After user log in to MVP portal, they can submit an application form for the MVP application

## Motivation
Create MVP submission form

## How Has This Been Tested?
Log in using your Okta id, then proceed to /application page, and try to submit you application

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [x] I have read the Contributing guide.
- [x] My code follows the code style of this project.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [x] My change is a code change and it requires an update to the documentation.
- [x] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

File: src/Project/MvpSite/rendering/appsettings.json [hunk 1]
```diff
@@ -3,7 +3,25 @@
...
+    "Username": "admin",
+    "Password": "b",
+    "MVPApplicationTemplateId": "5A6A420C-BE7E-4822-A887-33C83F5E9B0C",
+    "PersonTemplateId": "AD9C7837-8660-4360-BA2B-7ADDF4163685"
+  },
+  "Okta": {
+    "OktaDomain": "https://dev-45952888.okta.com",
+    "ClientId": "0oa1jhwqy63SWS4sG5d7",
+    "ClientSecret": "KrF9WOnKRCpYsQEjm-RPr9kJ-NjEHWpJBRyfpiUj",
+    "AuthorizationServerId": "default"
```
Comment (robearlam) [inline src/Project/MvpSite/rendering/appsettings.json:18]: Can we please remove these Okta details and change them to be populated by Environment variable.

File: MVP.Foundation.SitecoreExtensions/Pipelines/RegisterCustomRoute.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MVP.Foundation.SitecoreExtensions.Pipelines
+{
+    class RegisterCustomRoute
+    {
+    }
+}
```
Comment (robearlam) [inline MVP.Foundation.SitecoreExtensions/Pipelines/RegisterCustomRoute.cs:12]: Is this needed?

File: src/Feature/Forms/sitecore/ServicesConfigurator.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Mvp.Feature.Forms
+{
+    public class ServicesConfigurator : IServicesConfigurator
+    {
+        public void Configure(IServiceCollection serviceCollection)
+        {
+        }
+    }
+}
```
Comment (robearlam) [inline src/Feature/Forms/sitecore/ServicesConfigurator.cs:14]: Is this needed?

**Review by robearlam — CHANGES_REQUESTED** [review]
Looks good a few small changes, a couple of empty classes that might not be required.

The big blocker are some hard-coded okta details, can these be changed to run from the env vars like everything else?

File: src/Project/MvpSite/rendering/appsettings.json (hunk 1 above)
Comment (ahmed-ae) [reply src/Project/MvpSite/rendering/appsettings.json]: removed

File: MVP.Foundation.SitecoreExtensions/Pipelines/RegisterCustomRoute.cs (hunk 2 above)
Comment (ahmed-ae) [reply MVP.Foundation.SitecoreExtensions/Pipelines/RegisterCustomRoute.cs]: removed

File: src/Feature/Forms/sitecore/ServicesConfigurator.cs (hunk 3 above)
Comment (ahmed-ae) [reply src/Feature/Forms/sitecore/ServicesConfigurator.cs]: removed

**Review by robearlam — APPROVED** [review]

**Pull request merged**